<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Контрольная работа №1_2 (вариант 1)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать массив</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обработать массив</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Длина одномерного массива:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="countLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Интервал значений массива от</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leftLineEdit">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rightLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>40</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>до</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="unsortTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Значения</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="sortTableWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>101</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Значения</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
